code/obj/artifacts/artifactprocs.dm

```diff
@@ -245,6 +254,28 @@
 			src.ArtifactStimulus(random_stimulus,random_strength)
 	return
 
+/obj/proc/artifact_music_act(atom/instrument, note, volume)
+	if (src.artifact.activated)
+		return
+	var/dist = GET_DIST(src, instrument)
+	if (dist > 6)
+		return
+	if (dist > 3 || volume < 15)
+		src.visible_message("[src] quietly hums.", "[src] quietly hums.")
+		return
+
+	var/datum/artifact_trigger/note/trigger = locate(/datum/artifact_trigger/note) in src.artifact.triggers
+	var/similarity = trigger.get_similarity(note)
+	switch (similarity)
```

Comment:
I added doc comment to clarify. Otherwise, feels slightly awkward to put defines for something that's only meant to be used in 2 places in the code I guess, idk